src/LabelNumeral.jl

```diff
@@ -55,6 +55,6 @@ isless(n1::LabelNumeral, n2::LabelNumeral) = getval(n1) < getval(n2)
 ## Arithmetic
 # Multiple argument operators
 for op in [:+, :-,:max, :min]
-    @eval ($op){T <: Integer}(ns::LabelNumeral{T}...) =
-        $(op)(map(n -> getval(n), ns)...) |> LabelNumeral{T}
+    @eval ($op){T <: Integer}(n1::LabelNumeral{T}, ns::LabelNumeral{T}...) =
+        $(op)(map(getval, n1, ns)...) |> LabelNumeral{T}
```

Comment:
This line is breaking a few test cases on Roman Numerals. I think we should get rid of getval and replace it will "Int" conversion so that it will be more standard. 

In this context should the code be? `$(op)(getval(n1), map(getval, ns)...)  |> LabelNumeral{T}` as map will be applicable only on the collection part.